.flake8

```diff
@@ -0,0 +1,13 @@
+[flake8]
+# Rule definitions: http://flake8.pycqa.org/en/latest/user/error-codes.html
+# D203: 1 blank line required before class docstring
+# F401: 'identifier' imported but unused
+# E402: module level import not at top of file
+# C901: too complex
+# W503: line break before binary operator
```

Comment:
We only have references to D203,W503,F401 here, so can drop the definitions of C901 and E402.

Reply:
I don't follow this, shall we talk?..

Reply:
Basically I'm just saying you can delete the lines which explain what C901 and E402 are.